File: 2022/01/1.pl
#!/usr/bin/perl

my $max=0;
my $sum=0;


while(<>)
{
    if($_=~/(\d+)/)
    {
	print "linje: $1\n";
	$sum+=$1;
    }
    else
    {
	print "tom linje, $sum\n";
	if ($sum > $max) { $max=$sum;}
	$sum=0;
    }

    
}

print "max: $max\n";

File: 2022/01/1.py
#!/usr/bin/python3

sum=0
max=0
with open('input') as f:
    for line in f:


        if (len(line)>1):
            v=int(line.strip())
            sum += v
        else:
            print(sum)
            if(sum > max):
                max=sum
            sum=0
print('max', max)            
